app/Views/Templates/components/actions/dropdown/item.blade.php

```diff
@@ -1,5 +1,5 @@
 @props([
-    'variant' => 'link', // Variant of the dropdown item: 'link' or 'border'
+    'variant' => 'link', // Variant of the dropdown item: 'link', 'border', 'nav-header', or 'nav-header-border'
```

Comment:
can we just keep the name the variants "header" and "border". the nav-XXX seems rather long and the border doesn't always have to be near a header :D ?

Reply:
PS: Should have looked at the file first. 

header
border
header-border


Should be fine